tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -3193,6 +3216,15 @@ Status TfTensorToTrtWeights(const Tensor& tensor, TrtWeightStore* weight_store,
     case DT_UINT16:
       CopyToTrtInt32Array<DT_UINT16>(tensor, dst);
       break;
+    case DT_UINT32:
```

Comment:
Please extend the test to cover newly added types: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc#L1473